internal/core/graphics/image.rs

```diff
@@ -541,6 +541,26 @@ pub struct LoadImageError(());
 ///
 /// let image = Image::from_rgba8_premultiplied(pixel_buffer);
 /// ```
+///
+/// ### Sending Image to a thread
+///
+/// `Image` is not [`Send`], because it uses internal cache that are local to the Slint thread.
+/// If you want to create image data in a thread and send that to slint, you can construct the
+/// [`SharedPixelBuffer`] in a thread, and send that to slint
```

Comment:
```suggestion
/// `Image` is not [`Send`], because it uses internal cache that are local to the Slint thread.
/// If you want to create image data in a thread and send that to slint, construct the
/// [`SharedPixelBuffer`] in a thread, and send that to Slint's UI thread.
```